server/mscalendar/availability.go

```diff
@@ -215,22 +219,22 @@ func (m *mscalendar) setStatusFromCalendarView(user *store.User, currentStatus s
 
 	if len(user.ActiveEvents) == 0 {
 		var err error
-		if currentStatus == model.STATUS_DND {
+		if currentStatus == busyStatus {
 			err = m.Store.StoreUserActiveEvents(user.MattermostUserID, remoteHashes)
 			if err != nil {
 				return "", err
 			}
 			return "User was already marked as busy. No status change.", nil
 		}
-		err = m.setStatusOrAskUser(user, model.STATUS_DND, events)
+		err = m.setStatusOrAskUser(user, busyStatus, events)
 		if err != nil {
 			return "", err
 		}
 		err = m.Store.StoreUserActiveEvents(user.MattermostUserID, remoteHashes)
 		if err != nil {
 			return "", err
 		}
-		return "User was free, but is now busy. Set status to DND.", nil
+		return "User was free, but is now busy. Set status to busy.", nil
```

Comment:
Can you inject the `busyStatus` variable here? As well as the rest of the strings mentioning `set status to busy`. Also keep in mind these are debug strings, and only show up when using `/availability` as opposed to the scheduled status sync job.